Makefile

```diff
@@ -24,30 +25,24 @@ build: clean deps
 	pip uninstall --yes openfisca-country-template
 	find dist -name "*.whl" -exec pip install --force-reinstall {}[dev] \;
 
-check-syntax-errors:
-	python -m compileall -q .
-
-format-style:
+format:
 	@# Do not analyse .gitignored files.
 	@# `make` needs `$$` to output `$`. Ref: http://stackoverflow.com/questions/2382764.
-	ruff format `git ls-files | grep "\.py$$"`
-	isort `git ls-files | grep "\.py$$"`
 	black `git ls-files | grep "\.py$$"`
```

Comment:
Why keeping Black and isort when Ruff says "Drop-in parity with Flake8, isort, and Black" ?

Reply:
For isort, because of heavy customisation. But black can go, en effect.